### DISCUSSION TITLE
`UbiquitousStorageKey`

### DISCUSSION THREAD

**Discussion by lucamegh** [pointfreeco/swift-sharing#22] [Ideas]
Introducing a persistent strategy that uses `NSUbiquitousKeyValueStore` (`UbiquitousStorageKey`, maybe?) could be a valuable addition to the library. What do you think?

**Comment by mbrandonw** [comment]
Hi @lucamegh, I think that could be a powerful persistence strategy to have, but we would love if someone (like you!) could first incubate the idea as a separate library. We can link to it from the readme of swift-sharing, and maybe after some time of being out in the wild, and after we get a chance to experiment with it, we could bring it into the library properly. How does that sound?

**Reply by lucamegh** [reply]
Sounds great! I’ll keep you updated as soon as I have something to share.
